### PR TITLE
Add some attributes that should be ignored by default...

### PR DESCRIPTION
**Pull request by roesslerj**
Please discuss and if you agree, include in release.
Anything else to add?

### CODE REVIEW

**Review by diba1013 — COMMENTED** [review]
First of all, changelog (i.e. DoD). I assume, we have a test that loads this file and checks if there are invalid filter rules defined?

File: src/test/java/DefaultRecheckIgnoreTest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
+import de.retest.recheck.review.ignore.io.ErrorHandlingLoader;
+
+class DefaultRecheckIgnoreTest {
+
+	@Test
+	void default_recheck_ignore_should_load_without_exception() throws Exception {
+		final ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
```
Comment (diba1013) [inline src/test/java/DefaultRecheckIgnoreTest.java]: We have an [extension](https://github.com/retest/recheck/blob/bd0f8ded6cb6cb4b98dcc466e5b622170a4db660/src/test/java/de/retest/recheck/review/workers/LoadAndSaveFilterWorkerTest.java#L29) that does this.

**Review by diba1013 — CHANGES_REQUESTED** [review]

File: src/test/java/DefaultRecheckIgnoreTest.java (hunk 1 above)
Comment (roesslerj) [reply src/test/java/DefaultRecheckIgnoreTest.java]: Wow, didn't know! Thanks

File: CHANGELOG.md [hunk 2]
```diff
@@ -31,6 +31,8 @@ Table of Contents
 
 ### Improvements
 
+* Add sensible default values to the recheck.ignore file that is installed.
```
Comment (diba1013) [inline CHANGELOG.md]: Shouldn't this be in the 1.6.0 section?

**Review by diba1013 — CHANGES_REQUESTED** [review]

**Review by diba1013 — APPROVED** [review]
I did some additional changes and approved so that this in the next release.

**Pull request merged**